website/pages/docs/advanced-topics/monitoring.md

```diff
@@ -56,30 +56,21 @@ In production, it is common to use an OpenTelemetry [collector](https://opentele
 
 ### OpenTelemetry and Datadog
 
-In this quick example we will show how to connect an open telemetry collector to Datadog via OpenTelemetry exporter.
+In this example we will show how to send OpenTelemetry traces from the CLI directly to a Datadog agent.
 
-Firstly, you will need to have an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) running either locally or as a gateway. Here is an example of running it locally with docker:
-
-```bash
-docker run  -p 4319:4319 -v $(pwd)/config.yml:/etc/otelcol-contrib/config.yaml otel/opentelemetry-collector-contrib:0.91.0
-```
-
-following is an example for OTEL collector `config.yml` to receive traces locally on 4318 and export them to Datadog:
+First, you will need to [enable OpenTelemetry ingestion on the Datadog Agent](https://docs.datadoghq.com/opentelemetry/interoperability/otlp_ingest_in_the_agent#enabling-otlp-ingestion-on-the-datadog-agent). Add the configuration below to your `datadog.yaml` file:
```

Comment:
I actually found it easier to use the environment variable when running the Datadog Agent. Here's my docker-compose setup:

```yaml
version: "3.0"
services:
  agent:
    image: gcr.io/datadoghq/agent:7
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - /proc/:/host/proc/:ro
      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
    environment:
      DD_API_KEY: redacted
      DD_SITE: "datadoghq.eu"
      DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT: "0.0.0.0:4318"
    ports:
      - "4318:4318"
```

Reply:
Makes sense, I was running the agent on my Mac so not the most common setup. I'll update the PR

Reply:
Updated to list the 2 options. I'm planning to overhaul this doc once I get OTEL working with other services

Reply:
Simplified the docs to point to the Datadog setup docs instead